nautobot_netbox_importer/diffsync/models/abstract.py

```diff
@@ -198,6 +201,22 @@ def create_nautobot_record(nautobot_model, ids: Mapping, attrs: Mapping, multiva
             record = nautobot_model(**ids, **attrs)
             record.clean()
             record.save()
+
+            # To keep track of original created date of a ChangeLoggedModel that is updated
+            # when `record.save()` and to create a ChangeObject with the import change
+            if issubclass(type(record), ChangeLoggedModel):
+                record.created = model_data["created"]
```

Comment:
Can we do this step before calling `record.save()` above so as to avoid saving the model twice?

Reply:
IIRC I noticed that the `created` record was updated with the execution date when `.save()` was executed, so I had to do after, I will double check

Reply:
That sounds familiar, yeah.

Reply:
Yes, from documentation (https://docs.djangoproject.com/en/3.2/ref/models/fields/#datefield) the field is populated with `save()` so or we overwrite the method or we do an edit afterwards. I believe that the mission of the plugin is more aligned with the second.